### ISSUE TITLE
Config file with IDE versions to check

### ISSUE THREAD

**Issue by d-wojciechowski** [ChrisCarini/intellij-platform-plugin-verifier-action#5]
**Is your feature request related to a problem? Please describe.**
I would like to have another option to specify IDE versions. I would like to change the action file only when the workflow itself should be changed, not only parameters (mostly not relevant for workflow itself). In that case, I would have a clean file change history.

**Describe the solution you'd like**
I would like to have an option to specify the location to config file with platforms to be validated with (json/yaml/properties/csv/txt, does not matter too much).

```
    - name: Verify Plugin on IntelliJ Platforms
      id: verify
      uses: ChrisCarini/intellij-platform-plugin-verifier-action@v0.0.2
      with:
        config-path : my/config/location/config.txt
```
So then example config might look like this
```
    ideaIC:2019.3
    ideaIU:2019.3
    pycharmPC:2019.3
    goland:2019.3
    clion:2019.3
    ideaIC:LATEST-EAP-SNAPSHOT
```

**Comment by ChrisCarini** [comment]
Hi @d-wojciechowski - thank you for the suggestion and details!

This makes sense, I've just made the needed change and released version `v1.0.3`. I've updated the docs (ie, `README.md`) with an example of this - Please give it a try and let me know if the docs are clear and if this is what you were looking for.

Thank you again for the suggestion!

**Comment by ChrisCarini** [comment]
@all-contributors please add @d-wojciechowski for idea
